classes/local/bigbluebutton/recordings/recording_action.php

```diff
@@ -0,0 +1,216 @@
+<?php
+// This file is part of Moodle - http://moodle.org/
+//
+// Moodle is free software: you can redistribute it and/or modify
+// it under the terms of the GNU General Public License as published by
+// the Free Software Foundation, either version 3 of the License, or
+// (at your option) any later version.
+//
+// Moodle is distributed in the hope that it will be useful,
+// but WITHOUT ANY WARRANTY; without even the implied warranty of
+// MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the
+// GNU General Public License for more details.
+//
+// You should have received a copy of the GNU General Public License
+// along with Moodle.  If not, see <http://www.gnu.org/licenses/>.
+
+/**
+ * The mod_bigbluebuttonbn/bigbluebutton/recordings/action.
+ *
+ * @package   mod_bigbluebuttonbn
+ * @copyright 2021 onwards, Blindside Networks Inc
+ * @license   http://www.gnu.org/copyleft/gpl.html GNU GPL v3 or later
+ * @author    Jesus Federico  (jesus [at] blindsidenetworks [dt] com)
+ */
+
+namespace mod_bigbluebuttonbn\local\bigbluebutton\recordings;
+
+use mod_bigbluebuttonbn\instance;
+
+defined('MOODLE_INTERNAL') || die();
+
+/**
+ * Collection of helper methods for handling recordings actions in Moodle.
+ *
+ * Utility class for meeting actions
+ *
+ * @package mod_bigbluebuttonbn
+ * @copyright 2021 onwards, Blindside Networks Inc
+ * @license   http://www.gnu.org/copyleft/gpl.html GNU GPL v3 or later
+ */
+class recording_action {
```

Comment:
I like this approach, even though it is still a broker or mediator. Although it may be an intermediate step before the final design. 

In such case, as it is a helper already called action, we could remove that name from the methods, so they are called as: `recording_action::import` instead of `recording_action::action_import` etc

Reply:
Yes, that's fine. Having a different class allow to keep things small and not confuse as per the purpose.